driver/ixsystems/common.py

```diff
@@ -15,17 +16,16 @@
 
 class TrueNASCommon(object):
 
-    VERSION = "1.0.0"
+    VERSION = "1.0.1"
```

Comment:
Should we VERSION this as 2.0.0 now that is a fairly major change?

Reply:
Yes, that makes more sense.

Reply:
PR updated.